src/dotnet-svcutil/lib/src/dotnet-svcutil-lib.csproj

```diff
@@ -19,7 +19,7 @@
     <IsPackable>true</IsPackable>
     <IsShipping>$(Ship_SvcUtilPackages)</IsShipping>
     <MajorVersion>2</MajorVersion>
-    <MinorVersion>1</MinorVersion>
+    <MinorVersion>2</MinorVersion>
```

Comment:
@HongGit, I would like to double confirm with you on the version here, I remember we talked about making it a preview release, right?

Reply:
Correct, we can make this 2.2.0preview1.

Reply:
Ok. Updated.